### DISCUSSION TITLE
Rebasing the Greasemonkey patch from 0.21 to 0.24 - loadPyodide unavailable when loading pyodide.js

### DISCUSSION THREAD

**Discussion by tubaman** [pyodide/pyodide#5067] [General]
Hey Pyodide Folks! I'm trying to rebase [my PR](https://github.com/pyodide/pyodide/pull/3597) from 0.21 to 0.24.  When loading pyodide.js, `loadPyodide` is an object that looks like `{version: "0.24.1"}`(stringified) instead of a function.  It looks like it broke for me between 0.23.3 and 0.24.0. I've played around with removing version from [exports](https://github.com/pyodide/pyodide/blob/0.24.1/src/js/pyodide.umd.ts#L2).  Then `loadPyodide` ends up being an empty object.  When I console.log `loadPyodide` right before the export above, it looks like a function as I expect.  When trying to load an unpatched Pyodide 0.24.1 in Tampermonkey, I have the same issue.  For context, when [loading a file like pyodide.js in Tampermonkey using @require](https://github.com/tubaman/pyodide/blob/v0.21.3-greasemonkey-tampermonkey-demo/user.js#L10), Tampermonkey inlines the file.  Any ideas on how to get `loadPyodide` to show up as a function?

**Comment by tubaman** [comment]
[SOLVED] It looks like loadPyodide is available at window.loadPyodide.

**Reply by hoodmane** [reply]
If you can use es6 modules, perhaps using `pyodide.mjs` instead of `pyodide.js` will work better for you.
